modules/appeals_api/lib/appeals_api/notice_of_disagreement_pdf_constructor.rb

```diff
@@ -66,6 +69,24 @@ def pdf_options
     # rubocop:enable Metrics/PerceivedComplexity
     # rubocop:enable Metrics/AbcSize
 
+    def merge_page(temp_path, output_path)
```

Comment:
Overrode here as a proof of concept that we will probably want to move to the BasePdfConstructor, but I didn't want to potentially mess with the HLR output without understanding it further.